### PR TITLE
fix weird links

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/trackingprofile-of-wcf.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Represents a configuration section for creating a subscription to workflow track
   
 |Element|Description|  
 |-------------|-----------------|  
-|[\<participants>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/participants.md)|A configuration element that contains all queries for a specific workflow identified by the `a HYPERLINK "http://msdn.microsoft.com/library/library/system.servicemodel.activities.tracking.configuration.profileworkflowelement.activitydefinitionid(VS.100).aspx" ctivityDefinitionId` property.|  
+|[\<participants>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/participants.md)|A configuration element that contains all queries for a specific workflow identified by the <xref:System.ServiceModel.Activities.Tracking.Configuration.ProfileWorkflowElement.ActivityDefinitionId> property.|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/trackingprofile-of-wcf.md]: the xref is best as displayProperty=nameWithType

File: docs/framework/configure-apps/file-schema/windows-workflow-foundation/trackingprofile.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ Represents a configuration section for creating a subscription to workflow track
   
 |Element|Description|  
 |-------------|-----------------|  
-|[\<participants>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/participants.md)|A configuration element that contains all queries for a specific workflow identified by the **a HYPERLINK "http://msdn.microsoft.com/library/system.servicemodel.activities.tracking.configuration.profileworkflowelement.activitydefinitionid(VS.100).aspx"ctivityDefinitionId** property.|  
+|[\<participants>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/participants.md)|A configuration element that contains all queries for a specific workflow identified by the <xref:System.ServiceModel.Activities.Tracking.Configuration.ProfileWorkflowElement.ActivityDefinitionId> property.|
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/windows-workflow-foundation/trackingprofile.md]: also best with nameWithType

File: docs/framework/winforms/controls/soundplayer-class.md [hunk 3]
```diff
@@ -45,4 +45,4 @@ The `SoundPlayer` class enables you to easily include sounds in your application
  [Controls to Use on Windows Forms](../../../../docs/framework/winforms/controls/controls-to-use-on-windows-forms.md)  
  Provides a complete list of Windows Forms controls, with links to information on their use.  
   
- Also see [HYPERLINK "http://msdn.microsoft.com/library/11bxex12(v=vs.110)" My.Computer Object](https://msdn.microsoft.com/library/11bxex12\(v=vs.110\)) or [My.Computer Object](https://msdn.microsoft.com/library/11bxex12\(v=vs.120\)).
+ Also see [My.Computer Object](../../../visual-basic/language-reference/objects/my-computer-object.md) or [My.Computer Object](../../../visual-basic/language-reference/objects/my-computer-object.md).
```
Comment (rpetrusha) [inline docs/framework/winforms/controls/soundplayer-class.md]: nit: "See also" is better than "Also see"
The two links are the same. One can be removed.

**Review by rpetrusha — APPROVED** [review]
Left a few comments and suggestions, @mairaw, but you can merge when you're ready.

File: docs/framework/configure-apps/file-schema/wcf/trackingprofile-of-wcf.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ Represents a configuration section for creating a subscription to workflow track
   
 |Element|Description|  
 |-------------|-----------------|  
-|[\<participants>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/participants.md)|A configuration element that contains all queries for a specific workflow identified by the `a HYPERLINK "http://msdn.microsoft.com/library/library/system.servicemodel.activities.tracking.configuration.profileworkflowelement.activitydefinitionid(VS.100).aspx" ctivityDefinitionId` property.|  
+|[\<participants>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/participants.md)|A configuration element that contains all queries for a specific workflow identified by the <xref:System.ServiceModel.Activities.Tracking.Configuration.ProfileWorkflowElement.ActivityDefinitionIddisplayProperty=nameWithType> property.|
```
Comment (pkulikov) [inline docs/framework/configure-apps/file-schema/wcf/trackingprofile-of-wcf.md]: Missing "?" in front of `displayProperty=nameWithType`.

Comment (mairaw) [reply docs/framework/configure-apps/file-schema/wcf/trackingprofile-of-wcf.md]: yep. Just noticed that. Thanks @pkulikov

**Pull request merged**